mmvb_backend/toy_ais/views.py

```diff
@@ -0,0 +1,107 @@
+import json
+from http import HTTPStatus
+
+from django.http import JsonResponse
+from django.utils.decorators import method_decorator
+from django.views import View
+from django.views.decorators.csrf import csrf_exempt
+from toy_ais.implementations import TOY_AIS
+
+
+# TODO: we will need to protect this view in the future, maybe using JWT?
```

Comment:
We will need general protection for *all* AI implementations, so I wouldn't treat it separately/specifically for toy AIs.

Reply:
This is slightly different because ai implementations is within the context of the API (drf) but the toy ais is a pure django app. I'm leaving this so we don't forget about it when the time comes